File: ProjektyZadania/Zadanie2/konwerter.cs
using System;

class Konwerter {
    static void Main() {
        Console.Write("Wybierz kierunek konwersji C lub F ");
        string kierunek = Console.ReadLine().ToUpper();
        Console.Write("Podaj temperaturę: ");
        double temp = Convert.ToDouble(Console.ReadLine());
        if (kierunek == "C") {
            double wynik = temp * 1.8 + 32;
            Console.WriteLine($"{temp}°C = {wynik}°F");
        } else if (kierunek == "F") {
            double wynik = (temp - 32) / 1.8;
            Console.WriteLine($"{temp}°F = {wynik:F2}°C");
        } else {
            Console.WriteLine("Nieznany kierunek konwersji");
        }
    }
}

File: ProjektyZadania/Zadanie2/konwerter.py
kierunek = input("Wybierz kierunek konwersji C lub F ").upper()
temp = float(input("Podaj temperaturę: "))
if kierunek == "C":
    wynik = temp * 1.8 + 32
    print(f"{temp}°C = {wynik}°F")
elif kierunek == "F":
    wynik = (temp - 32) / 1.8
    print(f"{temp}°F = {wynik:.2f}°C")
else:
    print("Nieznany kierunek konwersji")
